### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by rsmoz**

### CODE REVIEW

**Comment by rsmoz** [comment]
Thoughts, @CodaFi?

File: README.md [hunk 1]
```diff
@@ -6,4 +6,7 @@ At WWDC 2015, Apple announced their plans to open-source Swift. Specifically, th
 
 Bedrock is an attempt at building a 100% Swift open source framework that covers some of the types that one might want out of Foundation, UIKit, and AppKit. (This isn't a UI framework, but will have some types that are more suited to UIKit than Foundation, such as Color.)
 
-¯\\_(ツ)\_/¯
\ No newline at end of file
+¯\\_(ツ)\_/¯
+
+
+This project is maintained by @rsmoz and @CodaFi.
```
Comment (CodaFi) [inline README.md]: How about turning these into links?

Comment (rsmoz) [inline README.md]: 👍

**Comment by rsmoz** [comment]
👍?

**Comment by CodaFi** [comment]
:boat:

**Comment by rsmoz** [comment]
Boat it?

**Comment by rsmoz** [comment]
:ship: or :shipit: 

We have standards for this repository, damnit!

**Pull request merged**